modules/katamari/src/test/ts/atomic/api/fun/ThrottlerTest.ts

```diff
@@ -1,68 +1,72 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Assert, describe, it } from '@ephox/bedrock-client';
+import Promise from '@ephox/wrap-promise-polyfill';
 import * as Throttler from 'ephox/katamari/api/Throttler';
 
-UnitTest.asynctest('Throttler.adaptable', (success) => {
-  const data: string[] = [];
-  const throttler = Throttler.adaptable((value: string) => {
-    data.push(value);
-  }, 250);
+describe('atomic.katamari.api.fun.ThrottlerTest', () => {
 
-  throttler.throttle('cat');
-  throttler.throttle('dog');
-  throttler.throttle('elephant');
-  throttler.throttle('frog');
+  it('Throttler.adaptable', () => new Promise<void>((success) => {
+    const data: string[] = [];
+    const throttler = Throttler.adaptable((value: string) => {
+      data.push(value);
+    }, 250);
+
+    throttler.throttle('cat');
+    throttler.throttle('dog');
+    throttler.throttle('elephant');
+    throttler.throttle('frog');
 
-  setTimeout(() => {
-    Assert.eq('eq', [ 'frog' ], data);
-    throttler.throttle('frog-goose');
-    throttler.throttle('goose');
     setTimeout(() => {
-      Assert.eq('eq', [ 'frog', 'goose' ], data);
-      success();
+      Assert.eq('eq', [ 'frog' ], data);
```

Comment:
Missed a couple of `Assert` conversions in this file it seems.

Reply:
Weird.

Reply:
Fixed.